javascript-web/README.md

```diff
@@ -0,0 +1,40 @@
+# Usage
+### package.json
+```
+dependencies: {
+  "@gomomento/generated-types-webtext": "$current_version",
+  "grpc-web": "1.3.1",
+  "google-protobuf": "3.21.0"
+}
+```
+
+### Basic usage shape
+```typescript
+const authToken = getClientAuthToken_never_harcode_server_api_keys();
```

Comment:
it looks like eventually we will have to build a separate web client?

Reply:
Going to want to make separate distributions in order to have the appropriate trim dependencies.

Instead of copy pasting the whole thing we might end up making the momento client rely on an internal protocol implementation, which we could either bundle into assembly-packages (better) or require users to choose dependencies (worse).